documentation/asciidoc/microcontrollers/debug-probe/swd-connection.adoc

```diff
@@ -35,10 +37,13 @@ $ make -j4
 ----
 
 In a debug build you will get more information when you run it under the debugger, as the compiler builds your program with the information to tell GDB what your program is doing.
-
-See Chapter 6 of https://datasheets.raspberrypi.com/pico/getting-started-with-pico.pdf[Getting started with Raspberry Pi Pico] for more information.
 ======
 
+NOTE: For devices that are _not_ Raspberry Pis, a variant of GDB that can debug ARM processors is required. Use one of the following alternatives depending on your operating system and device:
+* On Linux devices, use `gdb-multiarch`.
+* On macOS devices, use `arm-none-eabi-gdb`.
+* On Windows devices, use `gdb` bundled with the Raspberry Pi VSCode extension.
```

Comment:
Maybe say use `arm-none-eabi-gdb` for both Windows and macOS, as the extension is just downloading that from Arm's website (https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads)?

Reply:
IMO console commands all belong in the getting-started guide, but I guess the site should be less wrong.